spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -340,7 +344,8 @@ public void testInDirectWriteToBigQueryWithDiffInSchemaAndModeCheck() {
   }
 
   @Test
-  public void testIndirectWriteToBigQueryWithDiffInSchemaNullableFieldAndDisableModeCheck() {
+  public void testIndirectWriteToBigQueryWithDiffInSchemaNullableFieldAndDisableModeCheck()
+      throws InterruptedException {
```

Comment:
test methods throw Exception, not it's subclasses

Reply:
fixed this